README.md

```diff
@@ -1118,6 +1118,8 @@ They can be overridden for testing purposes in [the same way as other settings](
   ```
 
   **Note**: If `settings.aws.profile` is not set, the setting will fallback to the "default" profile.
+  It is recommended to use something other than `default` for the `aws.config` profile name.
+  Using `default` may have unintended side effects for other AWS services running on the node, such as `aws-iam-authenticator`.
```

Comment:
In some ways, I'd include a note that having a `[profile default]` defined in the config might cause un-intended consequences. Imagine someone having 2 profiles defined, `default` and `kubelet`, in the config but had this set to `kubelet`, then they would still step on the issue but would still be following the advice.

In other words, I might adjust the wording to address the contents of `settings.aws.config` rather than the value of `settings.aws.profile`.

Reply:
Great point, thanks @nairb774! The issue really is more about having `[profile default]` in that file, rather than the `aws.profile` setting. Good call!

I'll get this updated to phrase the issue better shortly.